### ISSUE TITLE
[Plugin] Remove McMMO or add back the features.

### ISSUE THREAD

**Issue by Magenent** [EarthMC/Issue-Tracker#523]
Well I know that for the most part the double smelt and double drop will not be enabled again. However its not just this, all pvp mcmmo stuff is disabled along with salvage and acrobatics. The only thing that really effects the server is the potions part of mcmmo, the rest barely effects the server. So i'd say to re-enable a good chunk of the features of mcmmo or to just remove the plugin, after all it'd help with lag.

**Comment by watergod1234** [comment]
No. Keep them disabled. Grinding massive amounts shouldn’t give such an advantage like a high axe level does. Although this is a tad hypocritical because alchemy similar, but still enabled. Alchemy is different because it’s up to the user to use the pots correctly when axes for example is just grinding to win.

**Comment by alekb1** [comment]
I'd say reenable combat mcmmo. It adds a more dynamic combat system where different weapons are utilized in different strategies. Combat is stale rn, as it takes forever to destroy someones armor, and swords give an interesting fire/bleed duality.

**Comment by unown006** [comment]
> No. Keep them disabled. Grinding massive amounts shouldn’t give such a great advantage like high levels in PvP areas such as axes.

Well if a player invest a large amount of time to grind out somthing to get better in a game whitch only gives a advantage over those who do do not invest the time. This is called progression its somthing minecraft is lacking in at the moment.

 Although this is a tad hypocritical because alchemy is somewhat similar, but still enabled. However, alchemy is different because it’s up to the user to use the pots correctly while axes is just grinding to win.

Not really you still have to grind out aclamy however this isnt just axes this is any combat mcmmo

**Comment by watergod1234** [comment]
> Well if a player invest a large amount of time to grind out somthing to get better in a game whitch only gives a advantage over those who do do not invest the time. This is called progression its somthing minecraft is lacking in at the moment.
Although this is a tad hypocritical because alchemy is somewhat similar, but still enabled. However, alchemy is different because it’s up to the user to use the pots correctly while axes is just grinding to win.
Not really you still have to grind out aclamy however this isnt just axes this is any combat mcmmo

Off topic but aren’t you banned for hacking?

**Comment by unown006** [comment]
> > Well if a player invest a large amount of time to grind out somthing to get better in a game whitch only gives a advantage over those who do do not invest the time. This is called progression its somthing minecraft is lacking in at the moment.
> > Although this is a tad hypocritical because alchemy is somewhat similar, but still enabled. However, alchemy is different because it’s up to the user to use the pots correctly while axes is just grinding to win.
> > Not really you still have to grind out aclamy however this isnt just axes this is any combat mcmmo
> 
> Off topic but aren’t you banned for hacking?

Currently yes that doesn't however take anything away from my point

**Comment by watergod1234** [comment]
Knew it...

**Comment by unown006** [comment]
I hope so as your people were the ones who supplied evidence if you could call it that for my ban However Lets get back on topic about Mcmmo's

I believe they should be enabled and see no reason not to other than double drops/smelting as that has an effect on the Economy to raise the inflation of gold

**Comment by Steadlington** [comment]
Non-Combat Mcmmo.
Tree Feller - Disabled due to lag reasons (When I would mass cut down trees it would lag the server a lot)
Double smelt - Obvious why it is disabled.
Double drop - Yet again, obvious.
Alchemy - A very fun grind for players and pays off for doing so as it benefits the player who is willing to do such a grind.  

Combat Mcmmo
Swords - To be honest it would be fair to enable this one as it isn't as op as the others, the bleed effect can be countered with potions quite easily.
Axes - This one should stay disabled as it's incredibly prone to RNG and does a lot of damage.
Bows - This one is tricky because max rank is basically equal to one shotting someone in prot 4 (Do not enable). However, the daze (Which still works in pvp) is still enabled and shouldn't be disabled.
Unarmed - This one isn't even worth enabling. 
Taming - Not sure as to if this one is actually disabled but if it is then it should be enabled as the grind for this costs gold and should benefit a player who is willing to do such a grind. 

These are just my opinions on the current state of the skills and their impact/why they are disabled.

**Comment by prokillerr** [comment]
Saying unarmed isnt worth proves Just how little you know. Unlike weapon, punches can do most of their dmg even when spam hitting(actually tested) and their dmg can be raised to at least 7+(have done it to another server then stopped grinding, maybe could increase it more). Imagine the benefits, spam hit someone with 7 dmg over slow hitting someone with 10+fire that can be countered by a fire res pot

**Comment by unown006** [comment]
Ok 

Unarmed as prokiller stated is worth enabling as strength 2 plus the ability to increase damage and take someone's weapon away (if they haven't spent time in the skill) 

Bows/axes my whole point is it takes time to grind these so if players do the rewards are worth getting the only reason anyone would be against this is because they do not want to gain the advantage others would have by grinding which is called being lazy also another fact I am going to point out is No one on earth EMC Before Terra Nova had Full maxed combat Mcmmo's at all and on Terra Nova the same is true and its even harder to grind because of the natural mob spawning being so low

Also, combat Mcmmo's allow one thing for someone to specialize in combat just like any other mcmmo
I for one would like to know the staffs reasoning for why these skills are currently disabled by staff

**Comment by Steadlington** [comment]
"Saying unarmed isnt worth proves Just how little you know." I do enjoy civil conversation. It's funny that this comment is coming from someone who refuses to grind Mcmmo and isn't even on the scoreboard. 

Now, why unarmed isn't worth having on there is because it de-values the other weapon skills. Having someone disarm you removes your ability to fight back if you haven't done the grind. Secondly, you can zerg other players with nothing in your hands (You risk nothing), yet still do a significant amount of damage. Imagine a bunch of players with 1000 Unarmed hitting someone in prot 4, they risk nothing and can kill the guy in prot 4. Benefits, none really. 

"Bows/axes my whole point is it takes time to grind these so if players do the rewards are worth getting the only reason anyone would be against this is because they do not want to gain the advantage others would have by grinding which is called being lazy" - Unown006

Okay to begin with this, I have high combat Mcmmo skills and still believe you shouldn't be enabling these skills. I would personally benefit from all of these being enabled. Still, I have stated the reasons why they would make PvP unbalanced and would ruin the server's current META. A bow at max rank can near enough one shot someone in Prot 4, no amount of grinding is enough for that kind of power.

**Comment by unown006** [comment]
> "Saying unarmed isnt worth proves Just how little you know." I do enjoy civil conversation. It's funny that this comment is coming from someone who refuses to grind Mcmmo and isn't even on the scoreboard.
> 
> Now, why unarmed isn't worth having on there is because it de-values the other weapon skills. Having someone disarm you removes your ability to fight back if you haven't done the grind. Secondly, you can zerg other players with nothing in your hands (You risk nothing), yet still do a significant amount of damage. Imagine a bunch of players with 1000 Unarmed hitting someone in prot 4, they risk nothing and can kill the guy in prot 4. Benefits, none really.

First of all that won't happen Do you want to know why? People are lazy as proven in Earth EMC which has been up for two years and almost no one has farmed unarmed and combat mcmmo's in general Also It again benefits those who take the time to specialize in a skill explained above also if you hit someone with prot 4 without armor you will die 100% of the time (reference to your zerg statement)


> 
> "Bows/axes my whole point is it takes time to grind these so if players do the rewards are worth getting the only reason anyone would be against this is that they do not want to gain the advantage others would have by grinding which is called being lazy" - Unown006
> 
> Okay to begin with this, I have high combat Mcmmo skills and still believe you shouldn't be enabling these skills. I would personally benefit from all of these being enabled. Still, I have stated the reasons why they would make PvP unbalanced and would ruin the server's current META. A bow at max rank can near enough one shot someone in Prot 4, no amount of grinding is enough for that kind of power.


Do you as none of them are maxed so they must not be very high ;) Also trying to revalue a dead point by saying you took the time to grind something which you did will not help you Everyone who specializes in PvP would benefit from these skills that's the point 
Also about the bow There are several ways to combat this did you know unarmed has this special abbility called arrow deflect whitch Increasing your Unarmed level, increases your chance of deflecting an incoming arrow. Deflecting enemy arrows has a chance to hit the person who fired the arrow, the chance is much lower than deflecting an arrow though.

+0.05% per level until max at 50% at level 1000

Also, acrobatics has a skill which allows you to negate all damage taken 20% of the time 
So Bow's are still not op when 70% of the time they would do no damage also you can use things such as speed 2 and sidestep their attacks 

Also at the moment as I have a feeling you have been in very few fights overall is People have a really hard time killing one another in prot 4 without extra numbers (average skill) mcmmo fixes that very well even if both players have mcmmo its still skill based however there are extra abilities that each player can use

**Comment by alekb1** [comment]
Agree with unown except that the chances regarding arrow damage are added differently. So the acrobatics perk does it so you have a 20% chance of taking half damage. If you know statistics, you would know that when you culminate the chances, it would actually be 50% for no damage, and 10% for half damage, meaning that you would get hit by the full force 4/10 times and half the damage 1/10 times. Still pretty good.

**Comment by Steadlington** [comment]
"First of all that won't happen Do you want to know why? People are lazy as proven in Earth EMC which has been up for two years and almost no one has farmed unarmed and combat mcmmo's in general Also It again benefits those who take the time to specialize in a skill explained above also if you hit someone with prot 4 without armor you will die 100% of the time (reference to your zerg statement). " 

Referencing classic? TN has been up a few months and people already have 1k in some combat mcmmos. Having unarmed as a skill (With people that will go to the effort of training it) is incredibly overpowered for PvP as a whole. It's not about the grind being worth an overpowered reward if it makes the game shittier in general. Now, you say people won't go to the effort of training these skills but it's already been proven that people will train Mcmmo abilities even when their effects are PvE based. 

"Do you as none of them are maxed so they must not be very high ;) Also trying to revalue a dead point by saying you took the time to grind something which you did will not help you Everyone who specializes in PvP would benefit from these skills that's the point
Also about the bow There are several ways to combat this did you know unarmed has this special abbility called arrow deflect whitch Increasing your Unarmed level, increases your chance of deflecting an incoming arrow. Deflecting enemy arrows has a chance to hit the person who fired the arrow, the chance is much lower than deflecting an arrow though."

But everyone who doesn't pvp and those who join the server (Fresh) will not benefit from these things being here as the progression curve before you can even fight someone who has 1k is a long road. Albeit it's a nice challenge but most people don't want to log on and grind Mcmmo for hours only to be on par with people who have played longer. It's a mixed bag, playing longer will naturally accumulate levels but NEEDING levels is far different from wanting them. 

Having an OP Mcmmo combat ability slightly counter another is like putting lava on a fire pit and saying it put the fire out. Yeah, the fire goes out but it's still molten. The point is that enabling something else doesn't solve the first problem to begin with, if you don't get the RNG then you're fucked. 

"Also, acrobatics has a skill which allows you to negate all damage taken 20% of the time
So Bow's are still not op when 70% of the time they would do no damage also you can use things such as speed 2 and sidestep their attacks

Also at the moment as I have a feeling you have been in very few fights overall is People have a really hard time killing one another in prot 4 without extra numbers (average skill) mcmmo fixes that very well even if both players have mcmmo its still skill based however there are extra abilities that each player can use"

I've killed plenty of people on here, pvp'd a lot on other servers too. I'm not the one banned for hacking. Now, it's hard to kill someone in prot 4 if they're logging but otherwise it's fairly easy if you get to pot spam them. Nothing is skill based with Mcmmo, bringing RNG into the classic combat system isn't any indicator of skill. Although, Acrobatics should be enabled for damage reduction as it's not super over-powered and would provide an actual benefit for players.

**Comment by Keennoo** [comment]
NNONONONONONO boys what are you doing. MLGTerra in the oold days had OP unarmed and he was too op. So why don't we learn about our mistakes. If you don't know who Terra is it's becuase you're a toyfag. Or a post classic fag. uguguguguguhuh and also if you do return it bring back double gold as it'll make the economy have some money $$ and it'll bring some inflation and then it'll make me good since I AM NUMBER 1 REPAIR BEFORE YOU NERFED IT MY LIFE WAS WASTED

**Comment by Kyoris** [comment]
To be fair disarming is OP af, considering that you can literally just yoink whatever the other person has, making fighting much more annoying, and will result in hit-and-run robberies being possible (with no good counter), and that should remain disabled. The rest of the pvp abilities I don't have a real problem with.

**Comment by prokillerr** [comment]
"stealdington:... Coming from someone Who refuses to grind mcmmo and isnt even on the scoreboard".. What? For the past 2 months I hadnt realized that they set grinding from spawners back to disabled and that mobs could spawn in nether normally so I couldnt grind.Claiming that I refuse to grind and that me not being on the scoreboard proves you're full of shit. You(or someone else i dont remember) said that grinding for unarmed would give an unfair advantage against those with god gear but you forget that god gear needs grinding too? Only difference is one is permanent and the other can be lost upon death... But can be taken back easily cuz even a 1 week newbie knows how easy it is to get god gear in emc. My point is, the long grind it takes to get high unarmed balances the high rewards.oh and dont say youll bemefit from it cuz "you have them grinded high" cuz Last time i checked the highest unarmed score wasnt even near enough to start getting an advantage, everyone has slacked off to grinding it cuz very few know its advantages.
Tldr mcmmo pvp Stats are Just another advantage you need to grind to get Just like god gear, only difference is that one is permanent and the other is not but they're still equally balanced by one needing a long grind and the other a small grind.

**Comment by Steadlington** [comment]
It's almost like you don't read and understand the arguments put forth by everyone in this thread. I would benefit from the weapon skills, not unarmed. 

By default, Mcmmo disables XP from spawners. I would think about 2 months that you would realise that you weren't leveling. A little bit dense aren't you? Furthermore, you didn't realise that you get Mcmmo levels from nether mobs (That still naturally spawn), in all this time you didn't see or kill any? Hard to believe this. 

People have already stated their opinions on if the grind should be worth the reward and I have stated that I don't believe it's worth enabling due to the reasons stated above. You say people have 'slacked off' to grinding it, but it's advantages are disabled? So, players will not grind for something that provides no benefit to begin with. Incredible logic.

**Comment by alekb1** [comment]
Anyway, the unarmed skill also gives the perk iron grip, which makes it so you have a chance to not loose your weapon, 100% at level 1000. So if you don’t want to loose your weapon, just train some unarmed!

**Comment by unown006** [comment]
> Agree with unown except that the chances regarding arrow damage are added differently. So the acrobatics perk does it so you have a 20% chance of taking half damage. If you know statistics, you would know that when you culminate the chances, it would actually be 50% for no damage, and 10% for half damage, meaning that you would get hit by the full force 4/10 times and half the damage 1/10 times. Still pretty good.

I did not know that however still not bad



> "First of all that won't happen Do you want to know why? People are lazy as proven in Earth EMC which has been up for two years and almost no one has farmed unarmed and combat mcmmo's in general Also It again benefits those who take the time to specialize in a skill explained above also if you hit someone with prot 4 without armor you will die 100% of the time (reference to your zerg statement). "
> 
> Referencing classic? TN has been up a few months and people already have 1k in some combat mcmmos. Having unarmed as a skill (With people that will go to the effort of training it) is incredibly overpowered for PvP as a whole. It's not about the grind being worth an overpowered reward if it makes the game shittier in general. Now, you say people won't go to the effort of training these skills but it's already been proven that people will train Mcmmo abilities even when their effects are PvE based.
> 

Your reference to some is one person with max swords everywhere else people are still very behind and unless you meet that one guy who has what you do not then you would be at a disadvantage however mcmmo's are just one of the many things you need to fight its just an extra step that most people do not take and yes I reference classic as History has, for the most part, repeated itself Also next to no one does nor likely will as a group as grinding mcmmo's taken a large amount of time and you can easily burn yourself out over it also many of the things that would make it manageable are not a thing anymore or broken. Also, believe it or not, to make progression rewarding you have to make it worth someone's time upon not doing so whatever it is, is worthless. Also about PvE skills most people do not grind out PvE skills because they already do the task on a regular basis and do not need to or it doesn't fit their playstyle which also goes into not everyone's playstyle is a PvP er some people don't care about the Mcmmo abilities which are the people you are defending by the way those people who don't take the time to specialize in combat are going to simply not be as good at combat then someone that has. 


> "Do you as none of them are maxed so they must not be very high ;) Also trying to revalue a dead point by saying you took the time to grind something which you did will not help you Everyone who specializes in PvP would benefit from these skills that's the point
> Also about the bow There are several ways to combat this did you know unarmed has this special abbility called arrow deflect whitch Increasing your Unarmed level, increases your chance of deflecting an incoming arrow. Deflecting enemy arrows has a chance to hit the person who fired the arrow, the chance is much lower than deflecting an arrow though."
> 
> But everyone who doesn't pvp and those who join the server (Fresh) will not benefit from these things being here as the progression curve before you can even fight someone who has 1k is a long road. Albeit it's a nice challenge but most people don't want to log on and grind Mcmmo for hours only to be on par with people who have played longer. It's a mixed bag, playing longer will naturally accumulate levels but NEEDING levels is far different from wanting them.
> 

Ok so take someone who has maxed out everything mcmmo wise than take someone who doesn't just because the player has mcmmo doesn't make them a god it just makes them harder to kill depending on what they do There are other things to consider like what weapon they use how they use it if they have prot 4 potions golden apples (or other regen effects) Also skill and lag are factors in this so you can have everything in the world but if a skilled player can either run you out of supplies of hard to get op combat gear like god apples, for instance, something that I doubt I'll ever have to fight more than once and sure you die or rather escape then what happens when you have the next encounter with that person? They suddenly they become a lot more killable now what about the regular easy stuff to get that they won't run out of being things like prot 4 armored and strength 2 speed 2 perhaps even god bows? Well you can either out skilled them in a fight out lag them in a fight or just don't fight alone as currently there are many hunting parties running around Terra Nova that do just that However if the player is geared skilled and has good ping and has mcmmo's and you do not with equal numbers you are simply at a disadvantage because they unlike you or whoever this argument would be for are at a disadvantage and that's there fault however just blaming mcmmo's because you think their op when it takes a long time months perhaps to grind them out is not the solution as you are defending people that don't take the time to specialize in it and again that's not bad its just that they will have to accept that they are putting themselves at a disavatage by doing so.

> Having an OP Mcmmo combat ability slightly counter another is like putting lava on a fire pit and saying it put the fire out. Yeah, the fire goes out but it's still molten. The point is that enabling something else doesn't solve the first problem to begin with, if you don't get the RNG then you're fucked.
> 

Are you defending or attacking my claim I can't tell so please clarify as Mcmmo's are sometimes due to luck as well 

> "Also, acrobatics has a skill which allows you to negate all damage taken 20% of the time
> So Bow's are still not op when 70% of the time they would do no damage also you can use things such as speed 2 and sidestep their attacks
> 
> Also at the moment as I have a feeling you have been in very few fights overall is People have a really hard time killing one another in prot 4 without extra numbers (average skill) mcmmo fixes that very well even if both players have mcmmo its still skill based however there are extra abilities that each player can use"
> 
> I've killed plenty of people on here, pvp'd a lot on other servers too. I'm not the one banned for hacking. Now, it's hard to kill someone in prot 4 if they're logging but otherwise it's fairly easy if you get to pot spam them. Nothing is skill based with Mcmmo, bringing RNG into the classic combat system isn't any indicator of skill. Although, Acrobatics should be enabled for damage reduction as it's not super over-powered and would provide an actual benefit for players.

lol bringing up a false ban because you can't defend your claim and yes combat logging and claim jumping is a problem I would like to address elsewhere but has nothing to do about mcmmo's that I am aware of About pot spamming poison gets countered by regen and so does a or several potions of harming as if the enemy you are attacking would/should have speed 2 you are very unlikely to do very much damage that won't get regened you would be much better off with a bow or melee however I will do more testing to confirm this claim. Also, the whole point of bringing in Mcmmo's is to bring a benifit for players also when I refer to the skill I refer to PvP skill.

**Comment by unown006** [comment]
> NNONONONONONO boys what are you doing. MLGTerra in the oold days had OP unarmed and he was too op. So why don't we learn about our mistakes. If you don't know who Terra is it's becuase you're a toyfag. Or a post classic fag. uguguguguguhuh and also if you do return it bring back double gold as it'll make the economy have some money $$ and it'll bring some inflation and then it'll make me good since I AM NUMBER 1 REPAIR BEFORE YOU NERFED IT MY LIFE WAS WASTED

I fought MLG Terra killed him several times and made a motion tracking meme about it any way he did not have max unarmed and already stated to counter unarms drop ability you need unarmed yourself

**Comment by unown006** [comment]
> To be fair disarming is OP af, considering that you can literally just yoink whatever the other person has, making fighting much more annoying, and will result in hit-and-run robberies being possible (with no good counter), and that should remain disabled. The rest of the pvp abilities I don't have a real problem with.

Already stated unarmed can be 100% countered with iron grip

**Comment by unown006** [comment]
> "stealdington:... Coming from someone Who refuses to grind mcmmo and isnt even on the scoreboard".. What? For the past 2 months I hadnt realized that they set grinding from spawners back to disabled and that mobs could spawn in nether normally so I couldnt grind.Claiming that I refuse to grind because im not on the scoreboard proves you're full of shit. You(or someone else i dont remember) said that grinding for unarmed would give an unfair advantage against those with god gear but you forget that god gear needs grinding too? Only difference is one is permanent and the other can be lost upon death... But can be taken back easily cuz even a 1 week newbie knows how easy it is to get god gear in emc. My point is, the long grind it takes to get high unarmed balances the high rewards.oh and dont say youll bemefit from it cuz "you have them grinded high" cuz Last time i checked the highest unarmed score wasnt even near enough to start getting an advantage, everyone has slacked off to grinding it cuz very few know its advantages.
> Tldr mcmmo pvp Stats are Just another advantage you need to grind to get Just like god gear, only difference is that one is permanent and the other is not but they're still equally balanced by one needing a long grind and the other a small grind.

Very good reply

**Comment by unown006** [comment]
> It's almost like you don't read and understand the arguments put forth by everyone in this thread. I would benefit from the weapon skills, not unarmed.
> 
> By default, Mcmmo disables XP from spawners. I would think about 2 months that you would realise that you weren't leveling. A little bit dense aren't you? Furthermore, you didn't realise that you get Mcmmo levels from nether mobs (That still naturally spawn), in all this time you didn't see or kill any? Hard to believe this.
> 
> People have already stated their opinions on if the grind should be worth the reward and I have stated that I don't believe it's worth enabling due to the reasons stated above. You say people have 'slacked off' to grinding it, but it's advantages are disabled? So, players will not grind for something that provides no benefit to begin with. Incredible logic.

Perhaps However Some Terra Nova Nations require people in Military they assemble to grind Combat Mcmmo's Also I am not allowed to state who so do not ask also don't try to use that against me because if you had information that you were not allowed to specifically comment on you would do the same thing.

**Comment by prokillerr** [comment]
Steadingwhatever its not that I didnt realize for 2 months that I wasnt lvling up... It was that for 2 months I didnt know wether a bug disabled spawner lvlingup or admins.. And didnt know nether mobs spawned cuz I dont rly lurk there... Not my type of place, I thought it only had lava and nether fotresses anyways since I thought no mobs spawned... Not in the mood for more arguing.. I made my points nothing more to say.

**Comment by Magenent** [comment]
> Non-Combat Mcmmo.
> Tree Feller - Disabled due to lag reasons (When I would mass cut down trees it would lag the server a lot)
> Double smelt - Obvious why it is disabled.
> Double drop - Yet again, obvious.
> Alchemy - A very fun grind for players and pays off for doing so as it benefits the player who is willing to do such a grind.
> 
> Combat Mcmmo
> Swords - To be honest it would be fair to enable this one as it isn't as op as the others, the bleed effect can be countered with potions quite easily.
> Axes - This one should stay disabled as it's incredibly prone to RNG and does a lot of damage.
> Bows - This one is tricky because max rank is basically equal to one shotting someone in prot 4 (Do not enable). However, the daze (Which still works in pvp) is still enabled and shouldn't be disabled.
> Unarmed - This one isn't even worth enabling.
> Taming - Not sure as to if this one is actually disabled but if it is then it should be enabled as the grind for this costs gold and should benefit a player who is willing to do such a grind.
> 
> These are just my opinions on the current state of the skills and their impact/why they are disabled.

mate i don't know what you mean by obivous of double smelt, earthmc is mostly grinding nowadays especially for new players, double smelt and less so double drop rewarded people for grinding mcmmo and you would have ingot for ore trades so if you didn't grind them you could still gain more than you would normally

**Comment by Steadlington** [comment]
Smelting is a child skill of two skills, it takes the mean of the two skills to calculate it's level. Having double the gold de-values towns, nations.. etc. To counter that you may say that we should increase the price of these things, but even then your gold would still be worth less. 

Now, double smelt is only disabled for gold, same with salvaging. These two skills are extremely rewarding already with other ores and speed up the time to collect mass amounts of them significantly. As for double drop, it creates the same problem as double smelt.

**Comment by watergod1234** [comment]
> lol bringing up a false ban

It definitely was not a false ban. I fought you and I recall a certain instance when you were eating and hitting me with your first at the same time. Tested this and it’s impossible. Now, it’s easy to discredit my claim by saying it never happened. But it did. And you’re banned so it doesn’t matter.

**Comment by unown006** [comment]
> > lol bringing up a false ban
> 
> It definitely was not a false ban. I fought you and I recall a certain instance when you were eating and hitting me at the same time. Tested this and it’s impossible. Now, it’s easy to discredit my claim by saying it never happened. But it did. And you’re banned so it doesn’t matter.

If you would like to debate this here's my discord unown#9247 drop by some time ;)

**Comment by KarlOfDuty** [comment]
We will not be removing mcmmo.
